diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/StringTemplateFormatRule.kt

```diff
@@ -77,4 +82,8 @@ class StringTemplateFormatRule(private val configRules: List<RulesConfig>) : Rul
             }
         }
     }
+
+    private fun bracesCanBeOmitted(node: ASTNode) = !node.hasAnyChildOfTypes(CALL_EXPRESSION)
```

Comment:
What about other expressions that need to be placed in braces? E.g., what will your rule do with `"${foo as Foo}`?